src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -69,9 +69,16 @@
                     i18n-placeholder
                     id="token"
                     formControlName="token"
-                    readonly>
-          </textarea>
+                    readonly
+                    #token></textarea>
         </div>
+        <button class="btn btn-primary mb-3 float-right"
+                i18n-aria-label="Copy to Clipboard"
+                title="Copy to Clipboard"
+                i18n-title
+                (click)="copyToClipBoard(token)">
```

Comment:
According to `copyToClipBoard` token is not a string here (like defined in `BootstrapCreateModalComponent`), what's the right type than and when was it assigned?

Reply:
its seems to be referring to the element with #token in line 73 and its a HtmlElement

Reply:
This could actually be simplified by removing the argument from the method and simply getting the value from the FormControl.

Reply:
changed it to preexisting copy to clipboard button